subgraph/src/fleek-nfa.ts

```diff
@@ -20,12 +20,15 @@ import {
   ApprovalForAll,
   CollectionRoleGranted,
   CollectionRoleRevoked,
+  Controller,
+  Holder,
```

Comment:
Should we call this owner instead? Or token owner?

Reply:
If you are referring to Holder -> Owner, yes. That's the change in the other PR.
But controllers are different.

Reply:
Yes I was referring to Holder here too. If that's coming in another PR that's cool then.